### PR TITLE
New test scenario added to test the new changes

### PR DESCRIPTION
**Pull request by akulshre-qa**

### CODE REVIEW

**Review by mkundu1 — APPROVED** [review]

File: tests/util/fixture_fluent.py [hunk 1]
```diff
@@ -54,3 +54,32 @@ def load_mixing_elbow_case_dat(with_launching_container):
     )
     yield session
     session.exit()
+
+
+_peridoicrot_withpipes_case_filename = None
+
+
+@pytest.fixture
+def load_peridoicrot_withpipes_cas(with_launching_container):
```
Comment (dnwillia-work) [inline tests/util/fixture_fluent.py:63]:
```suggestion
def load_peridoicrot_withpipes_cas(with_launching_container):
```
```suggestion
def load_peridoic_rot_with_pipes_cas(with_launching_container):
```

File: tests/test_solvermode/test_boundaries.py [hunk 2]
```diff
@@ -76,3 +80,52 @@ def test_boundaries_elbow(load_mixing_elbow_mesh):
         ].turb_viscosity_ratio()
         == 4
     )
+
+
+@pytest.mark.integration
+@pytest.mark.setup
+def test_boundaries_peridoicrot(load_peridoicrot_withpipes_cas):
```
Comment (dnwillia-work) [inline tests/test_solvermode/test_boundaries.py:87]:
```suggestion
def test_boundaries_peridoic_rot(load_peridoic_rot_with_pipes_cas):
```

File: tests/test_solvermode/test_boundaries.py [hunk 3]
```diff
@@ -76,3 +80,52 @@ def test_boundaries_elbow(load_mixing_elbow_mesh):
...
+        open("tests\\test_solvermode\\boundaries_peridoicrot_expDict")
+    )
+    boundary_test = {}
+    boundary_tested = {}
+    for name, boundary in pysession.setup.boundary_conditions.items():
+        boundary_test[name] = boundary()
+    boundary_tested["val_1"] = boundary_test
+    TestCase().assertDictEqual(boundary_tested["val_1"], boundary_exp["val_1"])
+
+    boundary_test = {}
```
Comment (dnwillia-work) [inline tests/test_solvermode/test_boundaries.py:100]: I like writing
```suggestion
    boundary_test = dict()
```

File: tests/test_solvermode/test_boundaries.py [hunk 4]
```diff
@@ -76,3 +80,52 @@ def test_boundaries_elbow(load_mixing_elbow_mesh):
...
+    for boundary_type in pysession.setup.boundary_conditions.get_active_child_names():
```
Comment (dnwillia-work) [inline tests/test_solvermode/test_boundaries.py:101]: I wonder if it's worth testing the `get_active_child_names()` list?

**Review by dnwillia-work — APPROVED** [review]

File: tests/util/fixture_fluent.py (hunk 1 above)
Comment (akulshre-qa) [reply tests/util/fixture_fluent.py:63]: Sure Dan, will do that

File: tests/test_solvermode/test_boundaries.py (hunk 4 above)
Comment (dnwillia-work) [reply tests/test_solvermode/test_boundaries.py:103]: Yeah, I think it would be worth it.  It would be a test of the dependency logic right.

**Pull request merged**